overlord/managers_test.go

```diff
@@ -718,9 +718,15 @@ apps:
 }
 
 func (s *mgrsSuite) TestHappyRemoveWithQuotas(c *C) {
+	// TODO: we need a variant of this test which disables the memory cgroup
```

Comment:
Just question, is this not covered by the spread task, or should we also verify this through unit test optimally?

Reply:
Ah makes a lot of sense, thanks for the clarification!